dev/PushNotifications/PushNotificationManager.h

```diff
@@ -20,6 +28,8 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
     private:
         static bool IsChannelRequestRetryable(const winrt::hresult& hrException);
         static bool IsBackgroundTaskBuilderAvailable();
+        static void RegisterUnpackagedApplicationHelper(const winrt::guid& remoteId, _Out_ wil::unique_cotaskmem_string& unpackagedAppUserModelId);
+        static winrt::hresult CreateChannelWithRemoteIdHelper(const winrt::guid& remoteId, ChannelDetails& channelInfo);
```

Comment:
Don't put `private static` methods in the header. Instead, just declare them in the .cpp file as loose functions.